pandas/tests/frame/apply/test_frame_apply.py

```diff
@@ -1254,7 +1254,7 @@ def test_agg_reduce(self, axis, float_frame):
         # dict input with lists with multiple
         func = dict([(name1, ["mean", "sum"]), (name2, ["sum", "max"])])
         result = float_frame.agg(func, axis=axis)
-        expected = DataFrame(
+        expected = pd.concat(
```

Comment:
i think this is ok to change

Reply:
actually if you add sort=False doesn't this match the original?

Reply:
The issue is that DataFrame sorts in certain cases. There is no argument `sort` to DataFrame that can be set to False in this test.